index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>My Portfolio</title>
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+ integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+ crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="./index.html">Ashna Ali</a>
+      <img src="./Union.jpg" alt="menu" class="menu-icon">
+    </nav>
+  </header>
+    <main>
+ <section class="main-section">
+  <h2 class="name">I'm Ashna Ali <br>
+   Glad to see you!</h2>
+  <p class="description">
+    I'm a software developer! I can help you build a product , feature or website Look through some of my work and
+   experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+  <h3>LET'S CONNECT</h3>
+  <ul class="links">
+    <li><a title="twitter" href="https://twitter.com/Ashna_Ali1"><img src="./Twitter.jpg" alt="twitter"></a></li>
+    <li><a title="linkedIn" href="https://www.linkedin.com/in/ashna-ali-342151255/"><img src="./linkedin.jpeg" alt="linkedin"></a></li>
+    <li><a title="Medium" href="https://medium.com/Ashna_Ali1"><img src="./Medium.jpg" alt="medium"></a></li>
+   <li><a title="github" href="https://github.com/Ashnaali3255/Portfolio/tree/Mobile-Branch"><img src="./Github.jpg" alt="github"></a></li>
+   <li><a title="Angelist" href="https://angelist.com/Ashna_Ali1"><img src="./Angel.jpg" alt="angelist"></a></li>
```

Comment:
- [ ] to match the Figma design kindly add the left-bottom border radius to your headline section
![image](https://user-images.githubusercontent.com/89282221/211835977-c3913fbe-d58a-45cd-ba41-f10ae232016f.png)

Reply:
Fixed.

Reply:
> * [ ]  to match the Figma design kindly add the left-bottom border radius to your headline section
>   ![image](https://user-images.githubusercontent.com/89282221/211835977-c3913fbe-d58a-45cd-ba41-f10ae232016f.png)
@Abenezer-Tilahun 
Can you tell me How to do this, please?